addons/mail/static/src/js/basic_view.js

```diff
@@ -1,17 +1,29 @@
 odoo.define('mail.BasicView', function (require) {
 "use strict";
 
-var BasicView = require('web.BasicView');
+const BasicView = require('web.BasicView');
 
-var mailWidgets = ['mail_followers', 'mail_thread', 'mail_activity', 'kanban_activity'];
+const mailWidgets = ['kanban_activity'];
 
 BasicView.include({
     init: function () {
         this._super.apply(this, arguments);
+        const post_refresh = this._getFieldOption('message_ids', 'post_refresh', false);
+        this.chatterFields = {
+            hasActivityIds: this._hasField('activity_ids'),
+            hasMessageIds: this._hasField('message_ids'),
+            hasMessageFollowerIds: this._hasField('message_follower_ids'),
+            hasRecordReloadOnAttachmentsChanged: post_refresh === 'always',
+            hasRecordReloadOnMessagePosted: post_refresh === 'always' || post_refresh === 'recipients',
```

Comment:
Intent of `post_refresh === 'recipients'` is to reload only when message contains recipients (i.e. checked suggested recipients and/or partner mentions), but since this was a work-around on a flawed design in the 1st place, I'm totally fine with reload whenever there is a `post_refresh` that is set.

Maybe just check it is set, instead of exact values `'always'` or `'recipients'`?